pkgs/development/tools/metal-cli/default.nix

```diff
@@ -2,22 +2,29 @@
 
 buildGoModule rec {
   pname = "metal-cli";
-  version = "0.7.3";
+  version = "0.7.4";
 
   src = fetchFromGitHub {
     owner = "equinix";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-PsTQuEbo1ReWwZV4D14mEGsU99j49kleiL/6Xyk+g9s=";
+    sha256 = "sha256-muhHBUb5Ttj4n6fJzIJMqics5rKupeSBZAd4JxZUe64=";
   };
 
+  vendorSha256 = "sha256-F8d5i9jvjY11Pv6w0ZXI3jr0Wix++B/w9oRTuJGpQfE=";
+
+  subPackages = [ "cmd/metal" ];
+
   ldflags = [
     "-X github.com/equinix/metal-cli/cmd.Version=${version}"
   ];
 
-  vendorSha256 = "sha256-F8d5i9jvjY11Pv6w0ZXI3jr0Wix++B/w9oRTuJGpQfE=";
+  # Tests aren't worth running (Few & irrelevant)
```

Comment:
Is it few/irrelevant because of subPackages?

If it's few/irrelevant either way then please keep `doCheck = false;`

Reply:
The problem is, we have the bot doing upgrades, if upstream adds new tests, we won't notice them. I've found many packages with good tests from upstream disabled in nixpkgs. That is why I try to keep tests enabled. Despite not having tests.

Reply:
> Is it few/irrelevant because of subPackages?

Yes.

> If it's few/irrelevant either way then please keep `doCheck = false;`

Tests aren't failing because of subPackages restricting the tests. But the existing tests check for version only.

Reply:
> Is it few/irrelevant because of subPackages?
> 
> If it's few/irrelevant either way then please keep `doCheck = false;`

Also, to mimic upstream build: https://github.com/equinix/metal-cli/blob/main/Makefile#L53

The tests part is collateral effect, since in the test itself it tries to build the binaries (https://github.com/equinix/metal-cli/blob/main/tests/main_test.go#L31) and the plus the tests is not in the go expected path (alongside its code).